invenio/legacy/bibsort/engine.py

```diff
@@ -404,7 +403,12 @@ def sort_dict(dictionary, spacing=1, run_sorting_for_rnk=False, sorting_locale=N
                 locale.setlocale(locale.LC_ALL, sorting_locale + '.UTF8')
             except locale.Error:
                 write_message("Setting locale to %s is not working.. ignoring locale")
-        sorted_records_list = sorted(dictionary, key=dictionary.__getitem__, cmp=locale.strcoll, reverse=False)
+
+        def locale_cmp_list(first, second):
+            return locale.strcoll(' '.join(first), ' '.join(second))
```

Comment:
@ludmilamarian is there some other way how to do it? Shall we concatenate with some special `locale` word separator if exists?